### PR TITLE
Korean Translation

### PR DESCRIPTION
**Pull request by big1219**
Korean Translation

- [x] translate metadata
- [ ] translate white-paper

### CODE REVIEW

**Comment by mfts** [comment]
Hi @big1219, first of all, thank you for your submission.  

I noticed that you are replacing the original `white-paper.md` with your translated version. Please **don't delete** the original `white-paper.md` file, instead please create a folder in the root repository called `i18n` and within that another folder with your language ISO code `ko` (see below):
```
...
i18n/
  ko/
...
```
Then please place both the translated files `white-paper.md` and `metadata.yml` in the newly created folder.

**Comment by mfts** [comment]
@big1219 your translation looks great. The only part missing is the translation of the `metadata.yml` file. Please let me know if you need any help with that.

**Comment by mfts** [comment]
Hi @big1219 thank you for your translations work. The only file missing a Korean translation is the `metadata.yml` https://github.com/teaxyz/white-paper/pull/69/files#diff-8119a25eed95c54cb705261a7a81af1e871179ebe5cdc24bd753c6c7ecf0e104 

Let me know if there's anything I can help you with.

**Comment by big1219** [comment]
Hi ser

I need your help. What shall i do?
I do not know what wrong?
Thank you
-----Original Message-----

**Comment by jhheider** [comment]
I believe the `abstract:` key in your `metadata.yml` need to be translated: https://github.com/teaxyz/white-paper/pull/69/files

File: i18n/ko/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  tea incentivizes the maintenance of open-source by allowing network
+  participants to stake value against the packages they depend on and want to
+  secure. The tea protocol’s graph provides immutable package registration,
+  dependency requirements, package authenticity, and usage oracles to inform
+  the tea remuneration algorithm. Systematic inflation is distributed to all
+  packages based on that algorithm. If security or development issues are
+  found, developers can make claims supported by evidence against the package,
+  and slashing may occur. Members of the open-source community can review
+  packages for quality issues, and the protocol can respond to these reviews
+  by enacting proportional slashing events.
```
Comment (mfts) [inline i18n/ko/metadata.yml:21]: @big1219 please translate these sentences to English :)

**Review by mfts — CHANGES_REQUESTED** [review]
Changes requested in the `metadata.yml` file

**Review by Edoseptian17 — APPROVED** [review]

**Pull request closed without merging**